src/lib/plugin.js

```diff
@@ -157,6 +157,13 @@ module.exports = class PluginVirtual extends EventEmitter2 {
   }
 
   async sendRequest (message) {
+    // for backward compat with ilp-kit version < 2.1.0, copy `.custom` to `.data`.
+    // This will only work in connector-to-connector send_message rpc calls as implemented here in
+    // ilp-plugin-virtual. When sent over a FiveBellsLedger, like in five-bells-integration-test,
+    // message.data will be stripped again by ilp-plugin-bells, because it does not comply with
+    // the Message schema as defined by five-bells-shared.
+    message.data = message.custom
```

Comment:
you shouldn't modify an object parameter, because that will cause side effects in the function that called this one